cmd/cloud-api-adaptor/main.go

```diff
@@ -71,6 +71,7 @@ func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 			flags.StringVar(&tlsConfig.KeyFile, "cert-key", "", "cert key")
 			flags.BoolVar(&tlsConfig.SkipVerify, "tls-skip-verify", false, "Skip TLS certificate verification - use it only for testing")
 			flags.BoolVar(&disableTLS, "disable-tls", false, "Disable TLS encryption - use it only for testing")
+			flags.IntVar(&cfg.serverConfig.ProxyTimeout, "proxy-timeout", 3, "Maximum timeout in minutes for establishing agent proxy connection")
```

Comment:
How about using the GO's duration string here, instead of specifying a value in minutes?
https://pkg.go.dev/time#ParseDuration